index.js

```diff
@@ -60,13 +60,54 @@ animalToRemove.remove()
 
 /***** Deliverable 1 *****/
 function toggleColor(element) {
-    if (element.style.color === "green") {
+    if (element.style.color === "red") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
-}
+console.log("BigRich")}
+
+header.addEventListener('click', function(){
+
+    toggleColor(header)
+
+})
+
 
 /***** Deliverable 2 *****/
 
+const likeButton = document.body.querySelector('button.like-button')
+
+
+likeButton.addEventListener('click', function(){
+
+    likes.textContent = `${traveler.likes++} BEEF`
+    console.dir(likes)
+
+})
 /***** Deliverable 3 *****/
+
+const form = document.querySelector('form#new-animal-sighting-form')
+
+form.addEventListener('submit', function(event){
+    event.preventDefault()
+    console.log('Submitted')
+
+    console.dir(event.target)
+
+    const animalSpecies = event.target[0].value
+    const animalLink = event.target[1].value
+    const animalPhoto = event.target[2].value
+    const animalDesc = event.target[3].value
+
+    const newArticleObj = {
+        species: animalSpecies,
+        video: animalLink,
+        photo: animalPhoto,
+        description: animalDesc
+    }
+
+    renderAnimalSightingPost(newArticleObj)
+    form.reset()
+
+})
```

Comment:
Great job here! Again, I would create a separate function to be passed in as a callback function. I love your descriptive variable names! Although I think form can be a bit more specific, in case we end up having more forms on the page.

Great job assembling an object of the data!

Great job using the pre-existing function on line 110!

+++++++++ for resetting the form!! 👏 

My steps for solving this are as follows:

1. Find the sightingForm on the page and save it to a variable
2. add an event listener to the form element, listen for a `submit` event.
3. The submit event needs to be applied to an entire form element, not the submit button.
4. Create the `handleFormSubmission` callback function, which should be triggered when the form submits.

`handleFormSubmission` steps:

1. since we are submitting a form, we need to prevent the default action of the form, which is to make a post request and redirect to a specific URL. We do not want the page to refresh, we want to prevent that action and stay on the current page.
2. save all the data the user typed in from the form into descriptive variables.
3. assemble an object with key value pairs descriptive of the data they contain.
4. Pass in the animal data object to the `renderAnimalSightingPost` function

```
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);

function handleFormSubmission(e){
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species: species,
        videoLink: videoLink,
        photo: photo,
        description: description
    }
    renderAnimalSightingPost(newData);
}

sightingForm.addEventListener("submit", handleFormSubmission)
```

With ES6 notation

```
const sightingForm = document.querySelector(`form#new-animal-sighting-form`);

const handleFormSubmission = e => {
    e.preventDefault();
    const species = e.target["species"].value;
    const videoLink = e.target["animal-link"].value;
    const photo = e.target["animal-photo"].value;
    const description = e.target["sighting-description"].value;
    const newData = {
        species,
        videoLink,
        photo,
        description
    }
    e.target.reset()
    renderAnimalSightingPost(newData);
}

sightingForm.addEventListener("submit", handleFormSubmission)
```